inc/c_util/tarray.h

```diff
@@ -10,167 +10,38 @@
-                    result = MU_FAILURE;                                                                                                                        \
-                }                                                                                                                                               \
-                else                                                                                                                                            \
-                {                                                                                                                                               \
-                    /*Codes_SRS_TARRAY_02_008: [ TARRAY_ENSURE_CAPACITY(T) shall succeed, record the new computed capacity, and return 0. ]*/                   \
-                    ((TARRAY_TYPEDEF_NAME(T)*)tarray)->capacity = new_capacity;                                                                                 \
-                    ((TARRAY_TYPEDEF_NAME(T)*)tarray)->arr = temp;                                                                                              \
-                    result = 0;                                                                                                                                 \
-                }                                                                                                                                               \
-            }                                                                                                                                                   \
-        }                                                                                                                                                       \
-    }                                                                                                                                                           \
-    return result;                                                                                                                                              \
-}
-
 /*macro to be used in headers*/                                                                                     \
 #define TARRAY_TYPE_DECLARE(T)                                                                                      \
-    TARRAY_DEFINE_STRUCT_TYPE(T)                                                                                    \
-    THANDLE_TYPE_DECLARE(TARRAY_TYPEDEF_NAME(T))                                                                    \
+    /*TARRAY_DEFINE_STRUCT_TYPE(T)                                                                                   */ \
+    /*THANDLE_TYPE_DECLARE(TARRAY_TYPEDEF_NAME(T))                                                                   */ \
```

Comment:
commented out? Should this be removed or uncommented? Here and below

Reply:
These are just hints.

Just like the ones in tarray_ll.h. 

Now they are properly marked as such.